aten/src/ATen/function_wrapper.py

```diff
@@ -696,7 +698,7 @@ def to_return_type(arg, option):
 
 
 def create_generic(top_env, declarations):
-    # type: (TopEnvironment, List[FunctionOption]) -> List[OutputDeclaration]
+    # type: (TopEnvironment, List[FunctionOption]) -> Tuple[List[OutputDeclaration], List[Tuple[str, str]]]
```

Comment:
`Tuple[str, str]` is not a very descriptive type; it would be helpful to know what exactly is the extra stuff we're returning here.

Reply:
Sure, I'll change this to a NamedTuple.